Choreographer_30FPS/README.md

```diff
@@ -0,0 +1,55 @@
+Teapot
```

Comment:
be different name: Teapot is already there, and moreTeapot also there.  Probably a typo